### DISCUSSION TITLE
Suggest `"type": "module"` when running `npm init`

### DISCUSSION THREAD

**Discussion by jimmywarting** [npm/feedback#480] [CLI]
Right now it feels like we are in a transition face to go over from commonjs to ESM and there is a big mix of commonjs, esm or hybrid packages out there. Would feel better if you auto suggested `"type": "module"` when running `npm init` to rush things a bit

**Comment by ljharb** [comment]
It wouldn't be; see https://github.com/npm/rfcs/issues/347
